services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/assign_a_new_activity.tsx

```diff
@@ -111,48 +143,74 @@ const AssignANewActivity = ({ numberOfActivitiesAssigned, showDiagnosticBanner }
+          tooltipTriggerTextClass='data-table-row-section topic-section'
+          tooltipTriggerTextStyle={topicsStyle}
+        />
+      ),
+      dateReleased: <div className="date-released">{dateReleased}{newTag}</div>,
+      previewLink: <a className="preview-link" href={`/activity_sessions/anonymous?activity_id=${activity.id}`} rel="noopener noreferrer" target="_blank">Preview</a>,
+      select: <a className="quill-button secondary fun focus-on-light outlined select-suggested" href={`/assign/activity-library?activityClassificationFilters[]=evidence&search=${encodeURI(activity.name)}`}>Select</a>
+    }
+  })
+
+  const evidenceSection = activitiesToSuggest.length ? (
+    <React.Fragment>
       <h1 className="evidence-header">Browse the newest Reading for Evidence activities</h1>
-      <table className="data-table suggested-activities-table">
-        <tr className="data-table-headers">
-          <th className="data-table-header">Tool</th>
-          <th className="data-table-header activity-header">Activity</th>
-          <th className="data-table-header topics-header">Topics</th>
-          <th className="data-table-header">Date released</th>
-        </tr>
-        <tbody className="data-table-body">
-          <div className="list-item">
-            {activitiesToSuggest.map((a) => {
-              return (
-                <tr className="data-table-row" key={a.id}>
-                  <span className="tool-col"><img alt={evidenceToolIcon.alt} src={evidenceToolIcon.src} /></span>
-                  <td className="data-table-row-section name-col">
-                    <a href={`/assign/activity-library?activityClassificationFilters[]=evidence&search=${encodeURI(a.name)}`}>{a.name}</a>
-                  </td>
-                  <td className="data-table-row-section topics-col">
-                    {a.topics.map((t, i) => {
-                      return <p key={i}>{t.join(" / ")}</p>
-                    })}
-                  </td>
-                  <td className="date-col">{a.publication_date}</td>
-                  <td className="preview-col"><a href={`/activity_sessions/anonymous?activity_id=${a.id}`} rel="noopener noreferrer" target="_blank">Preview</a></td>
-                  <button className="quill-button secondary medium focus-on-light outlined select-suggested" onClick={() => window.location.href = `/assign/activity-library?activityClassificationFilters[]=evidence&search=${encodeURI(a.name)}`} type="button">Select</button>
-                </tr>
-              )
-            })}
-          </div>
-        </tbody>
-      </table>
-    </div>
-  )
+      <DataTable
+        headers={headers}
+        rows={rows}
+      />
+      {showAllSuggestedActivities ? null : <button className="quill-button outlined small secondary focus-on-light show-all-button" onClick={handleClickShowAll} type="button">Show all {activitiesToSuggest.length} Reading for Evidence activities</button>}
```

Comment:
Does this need an `aria-label`?

Reply:
@eadams17 no, the text of the button functions as a label. `aria-label` is only necessary when the button doesn't contain anything that explains what it's doing (like if it's just an image, and the alt text. doesn't clarify the purpose of the button).